feature/about/src/main/java/io/github/droidkaigi/confsched2023/about/AboutScreen.kt

```diff
@@ -78,14 +87,12 @@ private fun AboutScreen(
         content = { padding ->
             LazyColumn(
                 Modifier
-                    .padding(padding),
+                    .padding(
```

Comment:
I modified the padding of AboutScreen a little
bacause unneccasary padding is passed to bottom like this.
<img src="https://github.com/DroidKaigi/conference-app-2023/assets/40256933/5e50b586-180d-4853-9b74-fa055cf2df7d" width="300">

If you know any other good idea, I'd like to hear them 🙏